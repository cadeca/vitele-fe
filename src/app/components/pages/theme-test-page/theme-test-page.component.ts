import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'wl-theme-test-page',
  templateUrl: './theme-test-page.component.html',
  styleUrls: ['./theme-test-page.component.scss']
})
export class ThemeTestPageComponent implements OnInit {

  constructor() { }

  ngOnInit(): void {
  }

}
