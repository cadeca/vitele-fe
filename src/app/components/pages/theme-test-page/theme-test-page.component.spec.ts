import {async, ComponentFixture, TestBed} from '@angular/core/testing';

import {ThemeTestPageComponent} from './theme-test-page.component';
import {TranslateModule} from '@ngx-translate/core';
import {MaterialModule} from '../../../material.module';

describe('ThemeTestPageComponent', () => {
  let component: ThemeTestPageComponent;
  let fixture: ComponentFixture<ThemeTestPageComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [
        TranslateModule,
        MaterialModule
      ],
      declarations: [ ThemeTestPageComponent ]
    })
      .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(ThemeTestPageComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
