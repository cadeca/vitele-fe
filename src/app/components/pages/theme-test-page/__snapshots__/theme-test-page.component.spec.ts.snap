// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeTestPageComponent should match snapshot 1`] = `
<wl-theme-test-page>
  <div
    class="theme-test-page"
  >
    <section>
      <div
        class="example-label"
      >
        Basic
      </div>
      <div
        class="example-button-row"
      >
        <button
          class="mat-focus-indicator mat-button mat-button-base"
          mat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Basic
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-button mat-button-base mat-primary"
          color="primary"
          mat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Primary
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-button mat-button-base mat-accent"
          color="accent"
          mat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Accent
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-button mat-button-base mat-warn"
          color="warn"
          mat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Warn
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-button mat-button-base mat-button-disabled"
          disabled="true"
          mat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Disabled
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <a
          aria-disabled="false"
          class="mat-focus-indicator mat-button mat-button-base"
          href="https://www.google.com/"
          mat-button=""
          tabindex="0"
          target="_blank"
        >
          <span
            class="mat-button-wrapper"
          >
            Link
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </a>
      </div>
    </section>
    <mat-divider
      aria-orientation="horizontal"
      class="mat-divider mat-divider-horizontal"
      role="separator"
    />
    <section>
      <div
        class="example-label"
      >
        Raised
      </div>
      <div
        class="example-button-row"
      >
        <button
          class="mat-focus-indicator mat-raised-button mat-button-base"
          mat-raised-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Basic
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-raised-button mat-button-base mat-primary"
          color="primary"
          mat-raised-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Primary
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-raised-button mat-button-base mat-accent"
          color="accent"
          mat-raised-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Accent
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-raised-button mat-button-base mat-warn"
          color="warn"
          mat-raised-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Warn
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-raised-button mat-button-base mat-button-disabled"
          disabled="true"
          mat-raised-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Disabled
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <a
          aria-disabled="false"
          class="mat-focus-indicator mat-raised-button mat-button-base"
          href="https://www.google.com/"
          mat-raised-button=""
          tabindex="0"
          target="_blank"
        >
          <span
            class="mat-button-wrapper"
          >
            Link
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </a>
      </div>
    </section>
    <mat-divider
      aria-orientation="horizontal"
      class="mat-divider mat-divider-horizontal"
      role="separator"
    />
    <section>
      <div
        class="example-label"
      >
        Stroked
      </div>
      <div
        class="example-button-row"
      >
        <button
          class="mat-focus-indicator mat-stroked-button mat-button-base"
          mat-stroked-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Basic
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-stroked-button mat-button-base mat-primary"
          color="primary"
          mat-stroked-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Primary
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-stroked-button mat-button-base mat-accent"
          color="accent"
          mat-stroked-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Accent
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-stroked-button mat-button-base mat-warn"
          color="warn"
          mat-stroked-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Warn
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-stroked-button mat-button-base mat-button-disabled"
          disabled="true"
          mat-stroked-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Disabled
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <a
          aria-disabled="false"
          class="mat-focus-indicator mat-stroked-button mat-button-base"
          href="https://www.google.com/"
          mat-stroked-button=""
          tabindex="0"
          target="_blank"
        >
          <span
            class="mat-button-wrapper"
          >
            Link
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </a>
      </div>
    </section>
    <mat-divider
      aria-orientation="horizontal"
      class="mat-divider mat-divider-horizontal"
      role="separator"
    />
    <section>
      <div
        class="example-label"
      >
        Flat
      </div>
      <div
        class="example-button-row"
      >
        <button
          class="mat-focus-indicator mat-flat-button mat-button-base"
          mat-flat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Basic
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-flat-button mat-button-base mat-primary"
          color="primary"
          mat-flat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Primary
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-flat-button mat-button-base mat-accent"
          color="accent"
          mat-flat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Accent
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-flat-button mat-button-base mat-warn"
          color="warn"
          mat-flat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Warn
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <button
          class="mat-focus-indicator mat-flat-button mat-button-base mat-button-disabled"
          disabled="true"
          mat-flat-button=""
        >
          <span
            class="mat-button-wrapper"
          >
            Disabled
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </button>
        <a
          aria-disabled="false"
          class="mat-focus-indicator mat-flat-button mat-button-base"
          href="https://www.google.com/"
          mat-flat-button=""
          tabindex="0"
          target="_blank"
        >
          <span
            class="mat-button-wrapper"
          >
            Link
          </span>
          <div
            class="mat-button-ripple mat-ripple"
            matripple=""
          />
          <div
            class="mat-button-focus-overlay"
          />
        </a>
      </div>
    </section>
    <mat-divider
      aria-orientation="horizontal"
      class="mat-divider mat-divider-horizontal"
      role="separator"
    />
    <section>
      <div
        class="example-label"
      >
        Icon
      </div>
      <div
        class="example-button-row"
      >
        <div
          class="example-flex-container"
        >
          <button
            aria-label="Example icon button with a vertical three dot icon"
            class="mat-focus-indicator mat-icon-button mat-button-base"
            mat-icon-button=""
          >
            <span
              class="mat-button-wrapper"
            >
              <mat-icon
                aria-hidden="true"
                class="mat-icon notranslate material-icons mat-icon-no-color"
                role="img"
              >
                more_vert
              </mat-icon>
            </span>
            <div
              class="mat-button-ripple mat-ripple mat-button-ripple-round"
              matripple=""
            />
            <div
              class="mat-button-focus-overlay"
            />
          </button>
          <button
            aria-label="Example icon button with a home icon"
            class="mat-focus-indicator mat-icon-button mat-button-base mat-primary"
            color="primary"
            mat-icon-button=""
          >
            <span
              class="mat-button-wrapper"
            >
              <mat-icon
                aria-hidden="true"
                class="mat-icon notranslate material-icons mat-icon-no-color"
                role="img"
              >
                home
              </mat-icon>
            </span>
            <div
              class="mat-button-ripple mat-ripple mat-button-ripple-round"
              matripple=""
            />
            <div
              class="mat-button-focus-overlay"
            />
          </button>
          <button
            aria-label="Example icon button with a menu icon"
            class="mat-focus-indicator mat-icon-button mat-button-base mat-accent"
            color="accent"
            mat-icon-button=""
          >
            <span
              class="mat-button-wrapper"
            >
              <mat-icon
                aria-hidden="true"
                class="mat-icon notranslate material-icons mat-icon-no-color"
                role="img"
              >
                menu
              </mat-icon>
            </span>
            <div
              class="mat-button-ripple mat-ripple mat-button-ripple-round"
              matripple=""
            />
            <div
              class="mat-button-focus-overlay"
            />
          </button>
          <button
            aria-label="Example icon button with a heart icon"
            class="mat-focus-indicator mat-icon-button mat-button-base mat-warn"
            color="warn"
            mat-icon-button=""
          >
            <span
              class="mat-button-wrapper"
            >
              <mat-icon
                aria-hidden="true"
                class="mat-icon notranslate material-icons mat-icon-no-color"
                role="img"
              >
                favorite
              </mat-icon>
            </span>
            <div
              class="mat-button-ripple mat-ripple mat-button-ripple-round"
              matripple=""
            />
            <div
              class="mat-button-focus-overlay"
            />
          </button>
          <button
            aria-label="Example icon button with a open in new tab icon"
            class="mat-focus-indicator mat-icon-button mat-button-base mat-button-disabled"
            disabled="true"
            mat-icon-button=""
          >
            <span
              class="mat-button-wrapper"
            >
              <mat-icon
                aria-hidden="true"
                class="mat-icon notranslate material-icons mat-icon-no-color"
                role="img"
              >
                open_in_new
              </mat-icon>
            </span>
            <div
              class="mat-button-ripple mat-ripple mat-button-ripple-round"
              matripple=""
            />
            <div
              class="mat-button-focus-overlay"
            />
          </button>
        </div>
      </div>
    </section>
    <mat-divider
      aria-orientation="horizontal"
      class="mat-divider mat-divider-horizontal"
      role="separator"
    />
    <section>
      <div
        class="example-label"
      >
        FAB
      </div>
      <div
        class="example-button-row"
      >
        <div
          class="example-flex-container"
        >
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a delete icon"
              class="mat-focus-indicator mat-fab mat-button-base mat-primary"
              color="primary"
              mat-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  delete
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a bookmark icon"
              class="mat-focus-indicator mat-fab mat-button-base mat-accent"
              color="accent"
              mat-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  bookmark
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a home icon"
              class="mat-focus-indicator mat-fab mat-button-base mat-warn"
              color="warn"
              mat-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  home
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a heart icon"
              class="mat-focus-indicator mat-fab mat-button-base mat-accent mat-button-disabled"
              disabled="true"
              mat-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  favorite
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
        </div>
      </div>
    </section>
    <mat-divider
      aria-orientation="horizontal"
      class="mat-divider mat-divider-horizontal"
      role="separator"
    />
    <section>
      <div
        class="example-label"
      >
        Mini FAB
      </div>
      <div
        class="example-button-row"
      >
        <div
          class="example-flex-container"
        >
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a menu icon"
              class="mat-focus-indicator mat-mini-fab mat-button-base mat-primary"
              color="primary"
              mat-mini-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  menu
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a plus one icon"
              class="mat-focus-indicator mat-mini-fab mat-button-base mat-accent"
              color="accent"
              mat-mini-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  plus_one
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a filter list icon"
              class="mat-focus-indicator mat-mini-fab mat-button-base mat-warn"
              color="warn"
              mat-mini-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  filter_list
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
          <div
            class="example-button-container"
          >
            <button
              aria-label="Example icon button with a home icon"
              class="mat-focus-indicator mat-mini-fab mat-button-base mat-accent mat-button-disabled"
              disabled="true"
              mat-mini-fab=""
            >
              <span
                class="mat-button-wrapper"
              >
                <mat-icon
                  aria-hidden="true"
                  class="mat-icon notranslate material-icons mat-icon-no-color"
                  role="img"
                >
                  home
                </mat-icon>
              </span>
              <div
                class="mat-button-ripple mat-ripple mat-button-ripple-round"
                matripple=""
              />
              <div
                class="mat-button-focus-overlay"
              />
            </button>
          </div>
        </div>
      </div>
    </section>
  </div>
</wl-theme-test-page>
`;
